sycl/test-e2e/Matrix/SG32/get_coord_bf16_gemm.cpp

```diff
@@ -0,0 +1,25 @@
+//==----------- get_coord_bf16_gemm.cpp  - DPC++ joint_matrix---------==//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+// REQUIRES: matrix
+
+// RUN: %{build} -o %t.out -DSYCL_EXT_ONEAPI_MATRIX_VERSION=4
+// RUN: %{run} %t.out
+// XFAIL: cpu, gpu
```

Comment:
The original test for sub-group 16 has // XFAIL: cpu, so I just added gpu for sub-group 32, since this part is not yet implemented for SG size = 32. If // XFAIL: cpu should be removed, it should be handled in a separate PR to not mix different topics.

Reply:
@Nuullll, why are we still setting xfail:cpu on the tests?
can we remove it now?

Reply:
> @Nuullll, why are we still setting xfail:cpu on the get_coord tests? can we remove it now?

Not now. We have to wait for the corresponding OCL CPU RT uplift.